### PR TITLE
Add technique Ensemble Learning

### PR DESCRIPTION
**Pull request by skoehnen**
Add the technique Ensemble Learning.

Adds an additional technique related to #215

### CODE REVIEW

File: instances/technique/ensembleLearning.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/EnsembleLearning",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Ensemble Learning' is a group of methods that allow the application of a combination of machine-learning methods and agggregate the results of those methods into one output.",
```
Comment (lzehl) [inline instances/technique/ensembleLearning.jsonld]: no capitalization if not needed (I assume this is not a name that is always capitalized)

File: instances/technique/ensembleLearning.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "k-means clustering",
```
Comment (lzehl) [inline instances/technique/ensembleLearning.jsonld]: this should be then "ensemble learning" no?

**Review by lzehl — CHANGES_REQUESTED** [review]

File: instances/technique/ensembleLearning.jsonld (hunk 1 above)
Comment (skoehnen) [reply instances/technique/ensembleLearning.jsonld]: The descriptions I have seen were always capitalized.

File: instances/technique/ensembleLearning.jsonld (hunk 2 above)
Comment (skoehnen) [reply instances/technique/ensembleLearning.jsonld]: Ups, yes you are right, I copied k-means to get a template.

Comment (skoehnen) [reply instances/technique/ensembleLearning.jsonld]: Fixed it.

File: instances/technique/ensembleLearning.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/EnsembleLearning",
```
Comment (lzehl) [inline instances/technique/ensembleLearning.jsonld]: this should be .../ensembleLearning

File: instances/technique/ensembleLearning.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/ensembleLearning.jsonld]: this should be "ensemble learning" within a sentence this is not capitalized (from what I see in the literature)

File: instances/technique/ensembleLearning.jsonld [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Ensemble Learning' is a group of methods that allow the application of a combination of machine-learning methods and agggregate the results of those methods into one output.",
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "Ensemble Learning",
```
Comment (lzehl) [inline instances/technique/ensembleLearning.jsonld]: this should be "ensemble learning" within a sentence this is not capitalized (from what I see in the literature)

File: instances/technique/ensembleLearning.jsonld (hunk 1 above)
Comment (lzehl) [reply instances/technique/ensembleLearning.jsonld]: here as sentence start "'Ensemble learning' ...

**Comment by skoehnen** [comment]
@lzehl I updated the capitalization.

File: instances/technique/ensembleLearning.jsonld (hunk 3 above)
Comment (lzehl) [reply instances/technique/ensembleLearning.jsonld]: @skoehnen the capitalization did not change...

File: instances/technique/ensembleLearning.jsonld (hunk 4 above)
Comment (lzehl) [reply instances/technique/ensembleLearning.jsonld]: @skoehnen the capitalization did not change...

**Review by lzehl — CHANGES_REQUESTED** [review]

**Comment by lzehl** [comment]
@skoehnen I've updated the capitalization now and made some small changes to the definition. Please double check and let me know if I can merge

**Review by lzehl — APPROVED** [review]

**Review by lzehl — APPROVED** [review]

**Pull request merged**